specification/Metadata/Semantics/README.md

```diff
@@ -26,6 +26,60 @@ For full usage see:
 * [`3DTILES_metadata`](../../../extensions/3DTILES_metadata/1.0.0) - 3D Tiles extension that assigns metadata to various components of 3D Tiles
 * [`EXT_feature_metadata`](https://github.com/CesiumGS/glTF/tree/master/extensions/2.0/Vendor/EXT_feature_metadata/1.0.0) - glTF extension that assigns metadata to features in a model on a per-vertex, per-texel, or per-instance basis
 
+<!-- omit in toc -->
+### **TILE_BOUNDING_BOX**
+
+The bounding volume of the tile, expressed as a [box (as defined by 3D Tiles 1.0)](https://github.com/CesiumGS/3d-tiles/tree/master/specification#box). This property may be used to describe a tighter bounding volume for a tile than is implicitly calculated by [3DTILES_implicit_tiling](https://github.com/CesiumGS/3d-tiles/blob/3d-tiles-next/extensions/3DTILES_implicit_tiling/0.0.0/README.md).
+
+* Type: `ARRAY`
+* Component type: `FLOAT32` or `FLOAT64`
+* Component count: `12`
+
+<!-- omit in toc -->
+### **TILE_BOUNDING_REGION**
+
+The bounding volume of the tile, expressed as a [region (as defined by 3D Tiles 1.0)](https://github.com/CesiumGS/3d-tiles/tree/master/specification#region). This property may be used to describe a tighter bounding volume for a tile than is implicitly calculated by [3DTILES_implicit_tiling](https://github.com/CesiumGS/3d-tiles/blob/3d-tiles-next/extensions/3DTILES_implicit_tiling/0.0.0/README.md).
+
+* Type: `ARRAY`
+* Component type: `FLOAT32` or `FLOAT64`
```

Comment:
Only `FLOAT64` should be allowed for regions.

Still determining whether we want to keep `FLOAT32` for bounding boxes, bounding spheres, and min/max heights. For now it's ok but `FLOAT32` should have a caveat attached to it since it's not the most precise way to represent bounding volumes at global scale. That could be a TODO for later once we go through https://github.com/CesiumGS/3d-tiles/pull/456.

Reply:
Added a note here https://github.com/CesiumGS/3d-tiles/pull/456#issuecomment-832742634